assets/charts/control-plane/kubelet/templates/kubelet-ds.yaml

```diff
@@ -35,6 +35,9 @@ spec:
           --authentication-token-webhook \
           --authorization-mode=Webhook \
           --client-ca-file=/etc/kubernetes/ca.crt \
+          {{- if .Values.cloudProvider }}
```

Comment:
We also needs the same for host kubelet. If it's done in the next commits (which I didn't see yet), commit message for this should be changed to mention that it only adds it for kubelet Helm chart.